.gitignore

```diff
@@ -19,5 +19,4 @@ secure.config
 ***.log
 ***.log.*
 ***.egg*
-.tox
```

Comment:
It is incorrect, revert this change please

Reply:
done